ext/hal/nordic/CMakeLists.txt

```diff
@@ -11,16 +11,23 @@ if(CONFIG_HAS_NRFX)
   zephyr_include_directories(nrfx/mdk)
   zephyr_include_directories(.)
 
+  # Connect Kconfig compilation option for Non-Secure software with option required by MDK/nrfx
```

Comment:
This is the same comments as "\# Define MDK defines globally."

Either the "Define MDK defines globally" comment should be improved, or this comment should be dropped, we don't want two comments saying the same thing in different ways. Because readers might assume the two comments are trying to say
different things.